--- a/vendor/mvdan.cc/sh/v3/interp/api.go
+++ b/vendor/mvdan.cc/sh/v3/interp/api.go
@@ -11,6 +11,7 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"math/rand"
 	"os"
 	"path/filepath"
@@ -310,11 +311,11 @@ func StdIO(in io.Reader, out, err io.Writer) RunnerOption {
 	return func(r *Runner) error {
 		r.stdin = in
 		if out == nil {
-			out = io.Discard
+			out = ioutil.Discard
 		}
 		r.stdout = out
 		if err == nil {
-			err = io.Discard
+			err = ioutil.Discard
 		}
 		r.stderr = err
 		return nil
diff --git a/vendor/github.com/sylabs/scs-library-client/client/pull.go b/vendor/github.com/sylabs/scs-library-client/client/pull.go
index 4e87092..25cd08d 100644
--- a/vendor/github.com/sylabs/scs-library-client/client/pull.go
+++ b/vendor/github.com/sylabs/scs-library-client/client/pull.go
@@ -9,6 +9,7 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"net/http"
 	"net/url"
 	"os"
@@ -186,7 +187,7 @@ type NoopProgressBar struct{}
 func (*NoopProgressBar) Init(int64) {}
 
 // ProxyReader is a no-op
-func (*NoopProgressBar) ProxyReader(r io.Reader) io.ReadCloser { return io.NopCloser(r) }
+func (*NoopProgressBar) ProxyReader(r io.Reader) io.ReadCloser { return ioutil.NopCloser(r) }
 
 // IncrBy is a no-op
 func (*NoopProgressBar) IncrBy(int) {}
diff --git a/vendor/github.com/sylabs/scs-library-client/client/restclient.go b/vendor/github.com/sylabs/scs-library-client/client/restclient.go
index 726fa3f..561743e 100644
--- a/vendor/github.com/sylabs/scs-library-client/client/restclient.go
+++ b/vendor/github.com/sylabs/scs-library-client/client/restclient.go
@@ -12,6 +12,7 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"net/http"
 	"net/url"
 
@@ -92,7 +93,7 @@ func (c *Client) commonRequestHandler(ctx context.Context, method string, path s
 		}
 		return []byte{}, fmt.Errorf("request did not succeed: http status code: %d", res.StatusCode)
 	}
-	objJSON, err = io.ReadAll(res.Body)
+	objJSON, err = ioutil.ReadAll(res.Body)
 	if err != nil {
 		return []byte{}, fmt.Errorf("error reading response from server:\n\t%v", err)
 	}
diff --git a/vendor/github.com/sylabs/scs-key-client/client/pks.go b/vendor/github.com/sylabs/scs-key-client/client/pks.go
index 0260671..6b8a622 100644
--- a/vendor/github.com/sylabs/scs-key-client/client/pks.go
+++ b/vendor/github.com/sylabs/scs-key-client/client/pks.go
@@ -9,7 +9,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"io"
+	"io/ioutil"
 	"net/http"
 	"net/url"
 	"strconv"
@@ -137,7 +137,7 @@ func (c *Client) PKSLookup(ctx context.Context, pd *PageDetails, search, operati
 		pd.Token = res.Header.Get("X-HKP-Next-Page-Token")
 	}
 
-	body, err := io.ReadAll(res.Body)
+	body, err := ioutil.ReadAll(res.Body)
 	if err != nil {
 		return "", fmt.Errorf("%w", err)
 	}
